Core/src/main/java/com/plotsquared/core/generator/HybridGen.java

```diff
@@ -65,7 +65,8 @@ private void placeSchem(
             int x,
             int z,
             boolean isRoad,
-            boolean isPopulating
+            boolean isPopulating,
+            boolean biomes
```

Comment:
With all the boolean parameters, I think it would be worth to have something like a `Feature` enum or something like that, and to pass (Enum)Sets around. That's more readable and allows to add more easily